slides-resources/05-scientific_applications.md

```diff
@@ -92,23 +92,21 @@ TA: Jenny Du
 <br/>
 - Analyzing these types of data can prove very challenging, given their sheer size and complexity.
 
+<!-- Question: Imagine you were training a model that contained information from a "chart". How would you incorporate the various text data? -->
+
 ---
 #### **DL as an analytical tool**
 - DL offers several advantages over alternative data analysis tools when dealing with large volumes of data:
   - Ability to model highly complex relationships
   - Ability to handle noisy data
   - Adaptability to diverse data types (e.g., text, images, time series)
   - Scalability
+- Given the complexity of human health, and the large amounts of health data now available, AI methods like DL have become increasingly popular in healthcare research $^{1}$
 
 <!-- Question: Why do we say that deep learning is "scalable"? What are classes of algorithms that do not "scale" as easily? -->
 
-
 ---
 #### **DL in healthcare research over the years**
-- Given the complexity of human health, and the large amounts of health data now available, AI methods like DL have become increasingly popular in healthcare research $^{1}$
-
-<!-- ---
-![Healtchare_publications](images/ai_healthcare_publications.png "Number of healthcare-related AI research publications between 2000-2021. Jimma, 2023") -->
```

Comment:
@ErikinBC is this not needed anymore?

Reply:
@ErikinBC on Slack: "I decided to use a different figure, but the reference is good to stay (I’ll remove the figure in the next PR)"